### PR TITLE
Schema.prisma

### PR DESCRIPTION
**Pull request by amyria3**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -10,31 +40,72 @@ datasource db {
 }
 
 model User {
-  id    String @id @default(uuid())
-  name  String
-  email String
-  List  List[]
+  userId              String @id @default(uuid())
```
Comment (dnmct) [inline prisma/schema.prisma:43]: why userId?
this is already on the user model?

Comment (dnmct) [reply prisma/schema.prisma:43]: same goes for the other properties

File: prisma/schema.prisma [hunk 2]
```diff
@@ -10,31 +40,72 @@ datasource db {
...
+  listId              String @id @default(uuid())
+
+  listName            String? @default("new list")
+
+  userIdentifier      String //entering correct identifier will Link to user
+  belongsTo           User @relation(fields: [userIdentifier], references: [userId])
+
+  items               Item[] ///opposite relation to userItem related to this List.
+
+  @@map("List")
```
Comment (dnmct) [inline prisma/schema.prisma:62]: why singular in all the other tables names you use plural

**Review by dnmct — CHANGES_REQUESTED** [review]
PLS don't use the modal name in the keys for a model...
that's just straight up redundant

**Comment by amyria3** [comment]
done (?) hopefully

**Review by dnmct — APPROVED** [review]

**Pull request merged**